lib/widgets/lightbox.dart

```diff
@@ -258,9 +262,28 @@ class _ImageLightboxPageState extends State<_ImageLightboxPage> {
     // The full image is available, so display it.
     if (frame != null) return child;
 
+    // Use `BoxFit.contain` if the original image size is larger than
+    // the device screen size, otherwise `null` (default).
```

Comment:
What's the behavior of the default? If we can it's probably best to pass an explicit value that specifies the behavior we want — if nothing else, that should make it easier to reason about this logic where we're choosing between different values for this option.